site/lists/creators.tree

```diff
@@ -115,4 +115,142 @@ Tom Preston-Werner
 Xavier Leroy
  wikipedia Xavier_Leroy
 Yukihiro Matsumoto
- wikipedia Yukihiro_Matsumoto
\ No newline at end of file
+ wikipedia Yukihiro_Matsumoto
+Acorn Computers	
+ wikipedia Acorn_Computers	
+Adele Goldberg	
+ wikipedia Adele_Goldberg_(computer_scientist)	
+Alain Colmerauer	
+ wikipedia Alain_Colmerauer	
+Alan Kay	
+ wikipedia Alan_Kay	
+Alfred Aho	
+ wikipedia Alfred_Aho	
+Anders Hejlsberg	
+ wikipedia Anders_Hejlsberg	
+Andrei Alexandrescu	
+ wikipedia Andrei_Alexandrescu	
+Anthony James Barr	
+ wikipedia Anthony_James_Barr	
+Bertrand Meyer	
+ wikipedia Bertrand_Meyer	
+Brad Cox	
+ wikipedia Brad_Cox	
+Brendan Eich	
+ wikipedia Brendan_Eich	
+Brian Fox	
+ wikipedia Brian_Fox_(computer_programmer)	
+Brian Kernighan	
+ wikipedia Brian_Kernighan	
+Chris Lattner	
+ wikipedia Chris_Lattner	
+Cleve Moler	
+ wikipedia Cleve_Moler	
+Vernon Reeves	
+ wikipedia COBOL	
```

Comment:
Hi Hassam(@hassamalhajaji),

Why are there six entries marked as COBOL?

Kind Regards,
Liam

Reply:
Hi Liam, @celtic-coder 

COBOL was a collaborative effort and unfortunately only one of the contributors, Jean Sammet, has a wikipedia page. I've redirected the rest to the COBOL wiki. I'll be adding official websites and github links soon so that should help. 

Best,
Hassam

Reply:
Lets' only add a Wikipedia link if the page is dedicated to that person.

Reply:
Understood!